pkg/kubectl/cmd/get/get_flags.go

```diff
@@ -159,7 +159,7 @@ func (f *PrintFlags) AddFlags(cmd *cobra.Command) {
 	f.CustomColumnsFlags.AddFlags(cmd)
 
 	if f.OutputFormat != nil {
-		cmd.Flags().StringVarP(f.OutputFormat, "output", "o", *f.OutputFormat, "Output format. One of: json|yaml|wide|name|custom-columns=...|custom-columns-file=...|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=... See custom columns [http://kubernetes.io/docs/user-guide/kubectl-overview/#custom-columns], golang template [http://golang.org/pkg/text/template/#pkg-overview] and jsonpath template [http://kubernetes.io/docs/user-guide/jsonpath].")
+		cmd.Flags().StringVarP(f.OutputFormat, "output", "o", *f.OutputFormat, "Output format. One of: json|yaml|wide|name|custom-columns=...|custom-columns-file=...|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=... See [custom columns](http://kubernetes.io/docs/user-guide/kubectl-overview/#custom-columns), [golang template](http://golang.org/pkg/text/template/#pkg-overview) and [jsonpath template](http://kubernetes.io/docs/user-guide/jsonpath).")
```

Comment:
No need to change that, there's no markdown support in bash, nor in the generated docs, iirc.

Reply:
@soltysh, you'll have to excuse me but I'm still newby on how the HTML code is generated. Isn't the markdown and HTML documentation for K8s (such as this: https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#get) is taking it's content from the output of these commands?
I've been working with @tengqm, @steveperry-53 and @jimangel from the documentation group to try and solve this [issue](https://github.com/kubernetes/website/pull/10778) stemming from this root cause.

Reply:
@soltysh I agree that the brackets doesn't mean a thing in the help message, but it could be interpreted as a illformed markdown link when we are generating HTML pages for it. In the long run, we need to fix the doc generators (go tool, docker image and js lib, messy ...). However, a quicker fix would be just remove these brackets to get the docs generated correctly.

Reply:
How about dropping the brackets or turn them into `<http://kubernetes.io/docs/user-guide/jsonpath>` similar to what majority of man pages does already.